samcli/lib/config/samconfig.py

```diff
@@ -190,6 +200,50 @@ def _write(self):
     def _version(self):
         return self.document.get(VERSION_KEY, None)
 
+    def _deduplicate_global_parameters(self, cmd_names, section, key, value, env=DEFAULT_ENV):
```

Comment:
Are we sure we want to do this?
Potentially erasing customer values.
I think that what is expected is to be able to define global values but also be able to supersede the global value by defining one in a more specific section.

Reply:
I think the global section should set default values that can be overwritten in specific sections.